validator/testdata/feature_tests/ads.out

```diff
@@ -166,6 +166,7 @@ PASS
 |          <option>plista</option>
 |          <option>polymorphicads</option>
 |          <option>popin</option>
+|          <option>pressboard</option>
```

Comment:
same here. No need to change this file.

Reply:
this change was added to pass the tests for the merge

Reply:
It's not required if you don't change validator/testdata/feature_tests/ads.html file.